src/tripCard/TripCard.unit.tsx

```diff
@@ -84,6 +86,7 @@ describe('TripCard component', () => {
   it('Should have only the Ladies Only icon', () => {
     const tripCard = shallow(<TripCard {...mockedProps} flags={{ ladiesOnly: true }} />)
     expect(tripCard.find(LadyIcon).exists()).toBe(true)
+    expect(tripCard.find(AloneInTheBackIcon).exists()).toBe(false)
     expect(tripCard.find(ComfortIcon).exists()).toBe(false)
     expect(tripCard.find(LightningIcon).exists()).toBe(false)
   })
```

Comment:
nit: add a test "Should have the aloneInTheBackIcon"

Reply:
There's no such test for Comfort or Lightning icons, only for Lady, which also tests the absence of the others. As they're just booleans, I found it "good enough".